fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/SessionFactory.java

```diff
@@ -20,17 +20,14 @@
 import static java.util.Objects.requireNonNull;
 import static org.slf4j.LoggerFactory.getLogger;
 
-import java.security.Principal;
```

Comment:
Is SessionFactory being retained? Should we rename it TransactionFactory at some point? (I guess that's what TransactionManager kind of is)

Reply:
It will be removed in the PR for https://jira.duraspace.org/browse/FCREPO-3073